applications/callflow/src/module/cf_dynamic_cid.erl

```diff
@@ -284,17 +298,17 @@ find_key_and_dest(ListJObj, Data, Call) ->
     case kz_json:get_value(<<"idx_name">>, Data) of
         'undefined' -> find_key_and_dest(ListJObj, Call);
         Idx ->
-            Groups = kapps_call:kvs_fetch('cf_capture_groups', Call),
+            Groups = get_callee_number('cf_capture_groups', Call),
```

Comment:
The spec for `get_callee_number` says it returns `ne_binary()` but Groups is being used as a JSON object. Can you update the spec to properly reflect this?